### ISSUE TITLE
Plugin rise 500 error

### ISSUE THREAD

**Issue by netolympus** [deecay/redmine_pivot_table#67]
I was using the previous version of the plugin (0.0.5) after updating to 0.0.7 when I try to access the pivot menu, I receive the "Redmine 500 error".
I have returned to the previous version (0.0.5), but the error persists in both versions.

In production log:
![error500](https://user-images.githubusercontent.com/54178570/63170017-96126400-c038-11e9-9aa3-fdb72f870ebf.JPG)

Could you please helpme?
I´m Using Redmine 2.6

**Comment by deecay** [comment]
Hi,

I think I know what the issue here is, but I have to look into how to address the issue. I assume this is about missing setting for max issues return.

**Comment by netolympus** [comment]
Hi deecay, in previous version I didnt set this parameter (textbox was empty). In new version it was fixed at 1000, and when I returned to 0.0.5 version, the value persist. Now, I set this value to 20 but problem still persist.

More information: If I leave the textbox of max issues empty, the error raise immediately, but if I set a value, for example 500, the error takes longer to appear (because the page takes longer to load). 

thanks in advance.
